frontend/src/layouts/AppLayout/AppLayout.tsx

```diff
@@ -425,6 +425,16 @@ export const AppLayout = ({ children }: LayoutProps) => {
                           </MenuItem>
                         </a>
                       </Link>
+                      <Link href={`/org/${currentOrg?.id}/user-secrets`} passHref>
+                        <a>
+                          <MenuItem
+                            isSelected={router.asPath === `/org/${currentOrg?.id}/user-secrets`}
+                            icon="system-outline-90-lock-closed"
```

Comment:
I re-used the "secret sharing" lottie here - in the future a new one would be better.